symforce/codegen/template_util.py

```diff
@@ -184,11 +184,15 @@ def render_template(
     )
 
     if autoformat:
-        rendered_str = filetype.autoformat(
-            file_contents=rendered_str,
-            template_name=template_path,
-            output_path=output_path,
-        )
+        try:
+            rendered_str = filetype.autoformat(
+                file_contents=rendered_str,
+                template_name=template_path,
+                output_path=output_path,
+            )
+        except:
+            print(rendered_str)
+            raise
```

Comment:
This is going to be a ton of output for lots of files, also this should use the logger  - maybe this should be `logger.debug`?  Should also have some header message about what this output is / where it's coming from

Reply:
Oh, oops. This wasn't supposed to be in here at all.

I always make this change when I'm debugging formatting induced syntax errors when making jinja template changes, but I normally change it back. Looks like I accidentally checked it in this time.

Unless for some reason you want this in here, I intend to remove it.

Reply:
> I always make this change when I'm debugging formatting induced syntax errors when making jinja template changes
I usually take this as a sign that it should get checked in behind some flag :)  I'm fine with whatever though

Reply:
Yeah... maybe my workflow is a bit crude. But I feel pretty content to stick with it, since adding it in is about as easy as setting a flag.

Maybe someday, but I say not this PR.